apps/web/src/data/de/license-data-short.ts

```diff
@@ -1,15 +1,67 @@
 import { LicenseData } from '@/data-types'
 
+const ccBaseLicence = {
+  title: 'Dieses Werk steht unter der freien Lizenz CC BY-SA __VERSION__.0',
+  shortTitle: 'CC BY-SA __VERSION__.0',
+  url: 'https://creativecommons.org/licenses/by-sa/__VERSION__.0/deed.de',
+  agreement:
+    'Mit dem Speichern dieser Seite versicherst du, dass du deinen Beitrag (damit sind auch Änderungen gemeint) selbst verfasst hast bzw. dass er keine fremden Rechte verletzt. Du willigst ein, deinen Beitrag unter der <a href="https://creativecommons.org/licenses/by-sa/__VERSION__.0/deed.de">Creative Commons Attribution/Share-Alike Lizenz __VERSION__.0</a> und/oder unter einer gleichwertigen Lizenz zu veröffentlichen, welche der Serlo Education e. V. entsprechend der Regelungen in den <a href="/terms">Nutzungsbedingungen</a> festlegen darf. Falls du den Beitrag nicht selbst verfasst hast, muss er unter den <a href="/terms">Nutzungsbedingungen</a> verfügbar sein und du stimmst zu, notwendigen Lizenzanforderungen zu folgen.',
```

Comment:
Will those infos be needed for the image-licenses as well at some point or did you just add it to make it fit the structure?

Reply:
Added it to make it fit the structure

Reply:
what's the upside to integrating it here?

Reply:
They fit well here, since this is where the other "licence data" is stored. It seems like a better option than hardcoding them in the component. 

If you have a suggestion for a better place I'm open for it :pray:

Reply:
My thoughts:
- since we don't use the same licenses for images and content I think they should have their own data structure/variable
- content licenses might move to the editor as well but until they do they are part of the frontend code, the image licenses are immediately part of the editor and might be used in the integrations as well so the code should already be in the editor imho

Reply:
Moved the licences to the code.